lib/datadog/tracing/contrib/grape/endpoint.rb

```diff
@@ -94,20 +95,27 @@ def endpoint_run(name, start, finish, id, payload)
 
                 span.set_error(payload[:exception_object]) if exception_is_error?(payload[:exception_object])
 
+                integration_route = endpoint.env['grape.routing_args'][:route_info].pattern.origin
+
                 # override the current span with this notification values
                 span.set_tag(Ext::TAG_ROUTE_ENDPOINT, api_view) unless api_view.nil?
                 span.set_tag(Ext::TAG_ROUTE_PATH, path)
                 span.set_tag(Ext::TAG_ROUTE_METHOD, request_method)
 
                 span.set_tag(Tracing::Metadata::Ext::HTTP::TAG_METHOD, request_method)
                 span.set_tag(Tracing::Metadata::Ext::HTTP::TAG_URL, path)
+
+                if Thread.current.key?(:datadog_http_routing) && Thread.current[:datadog_http_routing].is_a?(Array)
+                  Thread.current[:datadog_http_routing] << [:grape, endpoint.env['SCRIPT_NAME'], integration_route]
+                end
```

Comment:
```suggestion
                trace.set_tag(Tracing::Metadata::Ext::HTTP::TAG_ROUTE, endpoint.env['SCRIPT_NAME'] + integration_route)
```

Would this work instead?